### PR TITLE
Js1 week 1 homework samed ozcan

### PR DESCRIPTION
**Pull request by ozcansamed**

### CODE REVIEW

File: Week1/js-exercises/Exercise-1.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+console.log(helloWorldEnglish);
+console.log(helloWorldDutch);
+console.log(helloWorldTurkish);
+console.log(helloWorldGerman);
+console.log(helloWorldFrench);
+console.log(helloWorldSpanish);
+console.log(helloWorldPortuguese);
+console.log(helloWorldGreek);
+console.log(helloWorldArabic);
+console.log(helloWorldRussian);
```
Comment (nudge) [inline Week1/js-exercises/Exercise-1.js:27]: :+1:

File: Week1/js-exercises/Exercise-2.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+console.log('I'm awesome');
+it doesn't work and send us an error message.
+In JavaScript, we can escape a quote from considering it as an end of string quote,
+by placing a backslash (\) in front of the quote. */
+
+console.log('I"m awesome');
+/* Now it works, because of our escaping character, backslash(\) */
+//Or we can change the quotes :
+
+console.log("I'm awesome");
```
Comment (nudge) [inline Week1/js-exercises/Exercise-2.js:16]: :+1:

File: Week1/js-exercises/Exercise-3.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+'use strict';
+
+console.log('EXERCISE 3 - Log The Number');
+
+let numberX;
+console.log('The value of numberX is: undefined');
+console.log(numberX);
+numberX = 7;
+console.log('The value of myNumberX is: 7');
+console.log(numberX);
```
Comment (nudge) [inline Week1/js-exercises/Exercise-3.js:12]: :+1:

File: Week1/js-exercises/Exercise-4.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+'use strict';
+
+console.log('EXERCISE 4 - Log the String');
+
+let myString = 'Samed OZCAN';
+console.log('Samed OZCAN');
+console.log(myString);
+myString = 'I am still Samed OZCAN;)';
+console.log('I am still Samed OZCAN;)');
+console.log(myString);
```
Comment (nudge) [inline Week1/js-exercises/Exercise-4.js:12]: 👍

File: Week1/js-exercises/Exercise-5.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+'use strict';
+
+console.log('EXERCISE 5 - Round a number and log it');
+
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let myMaxNumber = Math.max(a, z);
+console.log(myMaxNumber);
```
Comment (nudge) [inline Week1/js-exercises/Exercise-5.js:12]: :+1:

File: Week1/js-exercises/Exercise-5.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+'use strict';
+
+console.log('EXERCISE 5 - Round a number and log it');
+
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let myMaxNumber = Math.max(a, z);
```
Comment (nudge) [inline Week1/js-exercises/Exercise-5.js:11]: Nice use of `Math.max()`!

File: Week1/js-exercises/Exercise-6.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+console.log('EXERCISE 6 - Log an array of animals');
+
+let myArray = [];
+console.log('[]');
+console.log(myArray);
+let animals = ['lion', 'rabbit', 'gazelle'];
+console.log(animals);
+animals.push('Piglet');
+console.log(animals);
```
Comment (nudge) [inline Week1/js-exercises/Exercise-6.js:13]: :+1:

File: Week1/js-exercises/Exercise-6.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+'use strict';
+
+console.log('EXERCISE 6 - Log an array of animals');
+
+let myArray = [];
+console.log('[]');
+console.log(myArray);
+let animals = ['lion', 'rabbit', 'gazelle'];
+console.log(animals);
+animals.push('Piglet');
```
Comment (nudge) [inline Week1/js-exercises/Exercise-6.js:12]: small nit: `Piglet` starts with an upper case letter where the other animals start with lower case.

File: Week1/js-exercises/Exercise-7.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+
+'use strict';
+
+console.log('EXERCISE 7 - Log the length of a string');
+
+let mySentence = 'Programming is so interesting!';
+console.log(mySentence);
+let lengthOfMySentence = mySentence.length;
+console.log(lengthOfMySentence);
+console.log('The length of mySentence is: ' + lengthOfMySentence);
```
Comment (nudge) [inline Week1/js-exercises/Exercise-7.js:11]: :+1:

File: Week1/js-exercises/Exercise-8.js [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+  console.log('SAME TYPE');
+} else {
+  console.log('Not the same...');
+}
+
+if (typeof myFirstObj === typeof mySecondObj) {
+  console.log('SAME TYPE');
+} else {
+  console.log('Not the same...');
+}
```
Comment (nudge) [inline Week1/js-exercises/Exercise-8.js:55]: :+1:

File: Week1/js-exercises/Exercise-9.js [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+let y = 21;
+y = y % 4;
+console.log('The value of y would be 1 after the calculation');
+console.log(y);
+
+let z = 13;
+z = z % 2;
+console.log('The value of z would be 1 after the calculation');
+console.log(z);
```
Comment (nudge) [inline Week1/js-exercises/Exercise-9.js:20]: :+1:

**Review by nudge — COMMENTED** [review]
Nice work 💯 !

File: Week1/js-exercises/Exercise-5.js (hunk 6 above)
Comment (ozcansamed) [reply Week1/js-exercises/Exercise-5.js:11]: Thank you so much :) You taught well...

**Pull request closed without merging**